finalPrep/annotations.js

```diff
@@ -3,40 +3,66 @@
 
```

Comment:
I saw the calendar images included, but did not see links to other final prep gists. You'll need to submit those in order to complete this assignment

As for your calendar- good start here. I would expect weekends to fill up more with study time or group project time. Also, while I admire having mid day time at the gym, I wonder about eating! Maybe that's something you can do on the way to, or back, but just something to think about, and I always encourage remaining flexible, and adjusting as you figure out what works for you.